docs/src/components/action/NpmCopyButton.tsx

```diff
@@ -80,17 +77,18 @@ export default function NpmCopyButton({
   return (
     <Button
       slots={{ root: StyledButton }}
```

Comment:
The use of Base UI is wrong, this React tree doesn't make sense:

- Base UI Button > Styled Button > Base UI Button > button

@michaldudak Should it be?

- Styled Button > Base UI Button > button

So:

```suggestion
    <StyledButton
```